documents/motions/02.2018.1 - Definitions.md

```diff
@@ -114,6 +114,10 @@ Within WCA we use the following definitions for terminology.
       <td>Senior Delegate</td>
       <td>A WCA Staff Member and Full Delegate who has been appointed by the WCA to manage a group of WCA Delegates.</td>
     </tr>
+    <tr>
+      <td>Senior Member</td>
+      <td>A member of one or more Committee(s) or Team(s) who has been a member in at least one of the Committee(s) Team(s) for a year and has been appointed by its Leader.</td>
```

Comment:
```suggestion
      <td>A Committee or Team Member who has been on it for at least a year and has been appointed by its Leader.</td>
```

I think this looks clearer. Also, in my opinion, a Senior member should be a per-Committee/Team role, so I suggested accordingly.